src/components/Timeline/Timeline.css

```diff
@@ -0,0 +1,233 @@
+/*----------------------------------------------
+    Timeline "Basic" & General style
+    timeline | date + text
+----------------------------------------------*/
+h1,h2,h3,h4,h5,h6 {
+  margin-top: 0;
+}
+
+.timeline {
+  --marker-line-color: #ccd5db;
+  --marker-dot-color: var(--orange);
+  line-height: 1.4em;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+.timeline h1, .timeline h2, .timeline h3, .timeline h4, .timeline h5, .timeline h6 {
+  line-height: inherit;
+}
+
+.timeline-item {
+  padding-left: 40px;
+  position: relative;
+}
+.timeline-item:last-child {
+  padding-bottom: 0;
+}
+
+.timeline-info {
+  font-size: 12px;
+  font-weight: 700;
+  letter-spacing: 3px;
+  margin: 0 0 0.5em 0;
+  text-transform: uppercase;
+  white-space: nowrap;
+}
+
+.timeline-marker {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  width: 12px;
+}
+.timeline-marker:before {
+  background: var(--marker-dot-color);
+  border: 3.2px solid transparent;
+  border-radius: 100%;
+  content: "";
+  display: block;
+  height: 12px;
+  position: absolute;
+  top: 4px;
+  left: 0;
+  width: 12px;
+  transition: background 0.3s ease-in-out, border 0.3s ease-in-out;
+}
+.timeline-marker:after {
+  content: "";
+  width: 2px;
+  background: var(--marker-line-color);
+  display: block;
+  position: absolute;
+  top: 26px;
+  bottom: 0;
+  left: 8px;
+}
+.timeline-item:last-child .timeline-marker:after {
+  content: none;
+}
+
+.timeline-item:not(.period):hover .timeline-marker:before {
+  background: transparent;
+  border: 3.2px solid var(--marker-dot-color);
+}
+
+.timeline-content {
+  padding-bottom: 40px;
+}
+.timeline-content p:last-child {
+  margin-bottom: 0;
+}
```

Comment:
I don't think here ":last-child" is useful, there's only one <p> inside timeline-content

Reply:
It could be useful if we want to have 2+ paragraphs so I'll keep it